pages/categorization.vue

```diff
@@ -350,17 +327,13 @@
 				// 0. Save a reference to the store table data set
 				let paintingData = this.$store.getters.paintingData;
 
-				// 0. Default row colors
-				let defaultBackgroundColor = this.$store.state.pageData.categorization.default.bColor;
-				let defaultForegroundColor = this.$store.state.pageData.categorization.default.fColor;
-
 				// 0. Counts number of painted columns
 				let columnCount = 0;
 
 				for ( let columnName in paintingData ) {
 
-					if ( defaultBackgroundColor != paintingData[columnName].bColor || 
-						 defaultForegroundColor != paintingData[columnName].fColor ) {
+					if ( this.defaultBackgroundColor != paintingData[columnName].bColor || 
```

Comment:
I really think we need to separate the categorization (and how we store categorization information in the store) from anything related to color, styling, and CSS. At the moment, if I understand things correctly, we have a global store called `pageData.categoriztion.paintingData` that keeps track of `tsvCategory` and `fColor` and `bColor` and `primaryKey` for the rows that have been annotated. 

And here we are counting columns that have been annotated with `countPaintedColumns` by comparing their color names to a default color name. But the color is just a visual guide, the important information is what category a column has been mapped to. So I really think that routing everything through the color adds a lot of hard to understand complexity here that we should think about clearing up before we merge this!